pkg/login/social/generic_oauth.go

```diff
@@ -27,6 +28,7 @@ type SocialGenericOAuth struct {
 	nameAttributePath    string
 	roleAttributePath    string
 	roleAttributeStrict  bool
+	groupsAttributePath  string
```

Comment:
Need to read this from config during initialization in `social.go`.

Reply:
thx. Added this